db/migrate/20220427155513_create_users.rb

```diff
@@ -1,6 +1,9 @@
 class CreateUsers < ActiveRecord::Migration[7.0]
   def change
     create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
 
       t.timestamps
     end
```

Comment:
- [ ] **[OPTIONAL]** It's always better to generate a new migration for adding new columns to an already existing table than to modify the table directly. What do you think? 😄